lms/templates/discussion/_thread_list_template.html

```diff
@@ -1,17 +1,6 @@
 <%page expression_filter="h"/>
 <%! from django.utils.translation import ugettext as _ %>
 <script type="text/template" id="thread-list-template">
-    <div class="forum-nav-header">
-        <button type="button" class="forum-nav-browse" id="forum-nav-browse" aria-haspopup="true">
```

Comment:
@andy-armstrong I agree: I'm also not keen that we use an icon that people associate with menus but not to signal a menu.